tests/page-object-models/home.page.ts

```diff
@@ -99,13 +99,10 @@ export class HomePage {
   async selectTestnet() {
     await this.page.getByTestId(SettingsSelectors.SettingsMenuBtn).click();
     await this.page.getByTestId(SettingsSelectors.ChangeNetworkAction).click();
+    await this.page.getByTestId(NetworkSelectors.NetworkListActiveNetwork).isVisible();
```

Comment:
I feel like this is not necessary, if you remove `click: force` on next line. playwright will wait until btn can be clicked

Reply:
Trying with it gone, though afaik the pseudo element causes issues and is why it was needed

Reply:
At the time I couldn't get the tests to pass without adding this but retrospectively it may have masked over another issue.